pkg/commands/version.go

```diff
@@ -33,7 +33,10 @@ func (e *Executor) initVersion() {
 	versionCmd := &cobra.Command{
 		Use:   "version",
 		Short: "Version",
-		RunE: func(cmd *cobra.Command, _ []string) error {
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 0 {
+				e.log.Fatalf("Usage: golangci-lint version")
```

Comment:
strange pattern https://github.com/golangci/golangci-lint/search?q=len%28args%29+%21%3D+0. why not `Args: cobra.NoArgs,` ?

Reply:
You are right, I rewrote all the commands.

Reply:
awesome!!!